src/server/pps/cmds/cmds.go

```diff
@@ -674,6 +675,80 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 	inspectPipeline.Flags().AddFlagSet(fullTimestampsFlags)
 	commands = append(commands, cmdutil.CreateAlias(inspectPipeline, "inspect pipeline"))
 
+	var editor string
+	var editorArgs []string
+	editPipeline := &cobra.Command{
+		Use:   "{{alias}} <pipeline>",
+		Short: "Edit the manifest for a pipeline in your text editor.",
+		Long:  "Edit the manifest for a pipeline in your text editor.",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) (retErr error) {
+			client, err := pachdclient.NewOnUserMachine("user")
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+		
+			pipelineInfo, err := client.InspectPipeline(args[0])
```

Comment:
Heads up that this will break when merged with `master` - `InspectPipeline` now takes a `details bool` arg (which this should set to true).

Reply:
@Tryneus I added the second parameter